api/include/opentelemetry/trace/span.h

```diff
@@ -74,13 +80,9 @@ class Span
   Span &operator=(const Span &) = delete;
   Span &operator=(Span &&) = delete;
 
-  // TODO
   // Sets an attribute on the Span. If the Span previously contained a mapping for
   // the key, the old value is replaced.
-  //
-  // If an empty string is used as the value, the attribute will be silently
```

Comment:
Why delete this comment?

Reply:
From the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#set-attributes):
> Attribute values expressing a numerical value of zero or an empty string are considered meaningful and MUST be stored and passed on to span processors / exporters.

Reply:
Ah, sorry, I misread it as empty key.